client/protocols/autostyle.py

```diff
@@ -15,6 +15,7 @@ class AutoStyleProtocol(models.Protocol):
 
     # Timeouts are specified in seconds.
     SHORT_TIMEOUT = 10
+    RETRY_LIMIT = 5
```

Comment:
Why is this here? Is it used anywhere? 

AFAIK it was removed in dd63e55c8fc2e9d62e412306a07479bf4796b8f1